src/compas_view2/objects/nurbssurfaceobject.py

```diff
@@ -0,0 +1,48 @@
+from compas.utilities.itertools import flatten
+from .meshobject import MeshObject
+
+
+class NurbsSurfaceObject(MeshObject):
+    """Object for displaying COMPAS NurbsSurface geometry."""
+
+    def __init__(self, surface, u=10, v=10, show_edges=False, **kwargs):
```

Comment:
to have the surface appear smooth we will need a higher default value. something like 50?